packages/block-library/src/navigation/block.json

```diff
@@ -50,6 +50,7 @@
 		"html": false,
 		"inserter": true,
 		"fontSize": true,
+		"__experimentalFontAppearance": true,
```

Comment:
Would it make sense to expand this to other blocks e.g: Site Title, Post Title, buttons?